modules/hosted-control-planes-version-support.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-version-support_{context}"]
+= Versioning support for hosted control planes
+
+//:FeatureName: Hosted control planes
+//include::snippets/technology-preview.adoc[]
+
+A few components of hosted control planes for {product-title} might require different versioning and support levels. The components include the HyperShift Operator, the Control Plane Operator, the command-line interface (CLI), and the API.
+
+The HyperShift Operator is released for each release of {product-title}. Any HyperShift Operator that is released for a minor _N_ release of {product-title} must support _N_, _N-1_, and _N-2_ minor releases of {product-title}.
```

Comment:
Should we give an example? HyperShoft operator for 4.12 should support 4.12, 4.11, and 4.10....

Reply:
Good idea! I'll add an example.

Reply:
I was just giving an example, it's tricky with real numbers. 

We could give a very futuristic number as well, but I see what you mean. Let's skip the example for now then.

Reply:
Okay, thanks. Yeah, I can't think of a way to provide an example without it sounding like hosted control planes are supported on a past version where they aren't actually supported, and we're not really supposed to refer to future releases in the docs. I can add a to-do for myself to update this topic with an example later when we can say something like, "...the HyperShift Operator for OCP version 4.13 also supports versions 4.12 and 4.11."